src/ontology/imports/chebi_imports.owl

```diff
@@ -74,7 +57,6 @@
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/CHEBI_23367"/>
         <obo:IAO_0000111 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">water</obo:IAO_0000111>
         <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">An oxygen hydride consisting of an oxygen atom that is covalently bonded to two hydrogen atoms.</obo:IAO_0000115>
-        <obo:IAO_0000412 rdf:resource="http://purl.obolibrary.org/obo/chebi.owl"/>
```

Comment:
This is the 'imported from' link, which we aren't currently adding with the new workflow. I kinda prefer to have these, because it's kinda useful if you're looking at `obi.owl`. If we do have these then the diff will be much smaller.